compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -359,21 +369,36 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 tcx.sess,
                 span,
                 E0599,
-                "no {} named `{}` found for {} `{}` in the current scope",
+                "no {} named `{}` found for {} `{}`{}",
                 item_kind,
                 item_name,
-                rcvr_ty.prefix_string(self.tcx),
+                if trait_missing_method {
+                    // FIXME(mu001999) E0599 maybe not suitable here because it is for types
+                    Cow::from("trait")
+                } else {
+                    rcvr_ty.prefix_string(self.tcx)
+                },
                 ty_str_reported,
+                if trait_missing_method {
+                    ""
+                } else {
+                    " in the current scope"
+                },
```

Comment:
Please also make these translatable too. `rustc_hir_typeck` currently has a lot of untranslatable diagnostics so it would be pretty nice to have PRs that touch code with untranslatable diagnostics to turn them translatable.